alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultProviderDataAccessor.java

```diff
@@ -196,69 +158,112 @@ public void mapUsersToProjectByEmail(String projectHref, Collection<String> emai
     }
 
     @Override
-    public void deleteUsers(DescriptorKey descriptorKey, Collection<ProviderUserModel> users) {
-        users.forEach(user -> providerUserRepository.deleteByProviderAndEmailAddress(descriptorKey.getUniversalKey(), user.getEmailAddress()));
-    }
-
-    @Override
-    public List<ProviderUserModel> saveUsers(DescriptorKey descriptorKey, Collection<ProviderUserModel> users) {
-        List<ProviderUserModel> savedUsers = users
-                                                 .stream()
-                                                 .map(user -> convertToUserEntity(descriptorKey, user))
-                                                 .map(providerUserRepository::save)
-                                                 .map(entity -> convertToUserModel(entity))
-                                                 .collect(Collectors.toList());
-        return savedUsers;
+    public void updateUserData(ProviderKey providerKey, Set<String> emailAddresses) {
+        updateUserDB(providerKey, emailAddresses);
     }
```

Comment:
Do we need the updateProjectDB method? If this is just a pass through, shouldnt we just move the content of the updateProjectDB method into updateUserData?

Reply:
This PR wasn't quite ready. I was gonna add you as a reviewer once it was done.